### PR TITLE
Add crane-operator bundle and index images

### PR DESCRIPTION
**Pull request by jmontleon**

### CODE REVIEW

File: ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.9.yaml [hunk 1]
```diff
@@ -1,12 +1,12 @@
 base_images:
   cli:
-    name: "4.9"
+    name: "4.10"
```
Comment (djzager) [inline ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.9.yaml:3]: This doesn't look like something we want to touch.

Comment (djzager) [reply ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.9.yaml:3]: Looks also to be the reason why it wasn't auto approved.

Comment (jmontleon) [reply ci-operator/config/openshift-kni/performance-addon-operators/openshift-kni-performance-addon-operators-release-4.9.yaml:3]: edited the wrong file :/

**Review by djzager — APPROVED** [review]
/lgtm

**Pull request merged**